### PR TITLE
Update delivery.yaml format and add docker base image as dependency

### PR DESCRIPTION
**Pull request by M4NTR4**
- delivery.yaml updated to new format
- add trigger on updated base_image

### CODE REVIEW

File: delivery.yaml [hunk 1]
```diff
@@ -1,26 +1,35 @@
...
 
-      npm install --no-optional
-      npm run dist
+pipeline:
+  - id: build_and_publish
+    type: script
+    commands:
+      - desc: Prepare
+        cmd: |
+          curl -o /tmp/ensure-docker -f https://delivery.cloud.zalando.com/utils/ensure-docker
```
Comment (scherniavsky) [inline delivery.yaml:14]: I guess that's a very old leftover and can be thrown away, as docker is built into CDP build VMs since long ago )

Comment (M4NTR4) [reply delivery.yaml]: Good point removed it

**Comment by M4NTR4** [comment]
:+1:

**Comment by scherniavsky** [comment]
:+1:

**Pull request merged**